### ISSUE TITLE
Batch normalization?

### ISSUE THREAD

**Issue by fzliu** [ethereon/caffe-tensorflow#17]
Awesome work, thanks for putting it together! With that said, are there any plans for supporting batch normalization?

**Comment by ethereon** [comment]
Thanks. Yes, batch normalization support is on my TODO list.

**Comment by Cospel** [comment]
This will be great!

**Comment by vodp** [comment]
@ethereon Thank you! I am really looking forward this feature.

**Comment by ethereon** [comment]
Batch normalization is now supported (tested on ResNet).

Caveat: Inference only. However, adding support for training should be fairly straightforward as well.
